README.md

```diff
@@ -152,6 +152,11 @@ When `[2]` is `nil`, then the real mapping has to be created by the `config()` f
       require("neo-tree").setup()
       end,
 }
+
+-- Example for vim-surround
+{
+    "tpope/surround", keys = { { 'S', mode = 'v' }, 'y', 'd', 'c' }
```

Comment:
wouldn't you make this `ys`, `ds` and `cs`

Reply:
Yeah, you could use that too I guess, but this what I've been using for years with zero problem.

Should I change it?

Reply:
🤷‍♂️ what I wrote is what I've been using and it only load the plugin when it's rly needed and not for example when you do `c$`

Reply:
Honestly, I'd rather have a lua plugin in there. I use zero vimscript pugins out of 96 plugins.

Just remove that part. I'll add some examples myself. Btw, a lof the code in the readme is auto-generated. Those examples are coming from the `examples.lua` file

Reply:
Cool 👍🏽